cost-analyzer/charts/prometheus/Chart.yaml

```diff
@@ -18,3 +18,7 @@ sources:
 - https://github.com/kubernetes/kube-state-metrics
 tillerVersion: '>=2.8.0'
 version: 11.0.2
+dependencies:
+  - condition: kube-state-metrics.disabled
+    name: kube-state-metrics
+    repository: file://./charts/kube-state-metrics
```

Comment:
@AjayTripathy After our metrics emission changes, do we still need to have KSM in the Helm chart?

Reply:
Probably not but that's a pretty big change outside the scope of this PR.